pkg/ovs/openflow/ofctrl_packetout.go

```diff
@@ -286,6 +289,13 @@ func (b *ofPacketOutBuilder) SetOutport(outport uint32) PacketOutBuilder {
 	return b
 }
 
+// SetL4Packet sets the L4 packet of the packetOut message. It provides a generic function to create a packet
+// of other protocols other than TCP/UDP/ICMP.
```

Comment:
maybe ... of protocol other other than TCP/UDP/ICMP

Reply:
updated.